CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Adds support for reading current state version outputs to StateVersionOutputs, which can be useful for reading outputs when users don't have the necessary permissions to read the entire state.
 * Adds Variable Set methods for `ApplyToWorkspaces` and `RemoveFromWorkspaces` [#375](https://github.com/hashicorp/go-tfe/pull/375)
+* Adds Run Tasks API support by @glennsarti [#381](https://github.com/hashicorp/go-tfe/pull/381), [#382](https://github.com/hashicorp/go-tfe/pull/382) and [#383](https://github.com/hashicorp/go-tfe/pull/383)
```

Comment:
I would split this entry into 3 for each run task change you made:
```markdown
* Add Run Task permissions field to `Organization` and `Workspace` by @glennsarti  [#383](https://github.com/hashicorp/go-tfe/pull/383)
* Add Run Task field to `Entitlement` struct  [#381](https://github.com/hashicorp/go-tfe/pull/381)
* Add `Enabled` field to `RunTask` struct [#382](https://github.com/hashicorp/go-tfe/pull/382)
```

Reply:
We haven't previously mentioned run tasks at all in the changelog.

Reply:
I think it would be OK to just say "Run tasks!" and I'm in favor of keeping the original version.

Reply:
It's listed in v0.26.0

Reply:
Ah! I was only looking in CHANGELOG. You're right. We did label it "beta" so this is still fine, I guess.